ngraph/frontend/tensorflow/src/op/interpolate.cpp

```diff
@@ -2,6 +2,9 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+// Please review the entire file. These changes have been merged without a detailed review.
+// This comment will be removed after that.
+
```

Comment:
line 39 Does input always 4D? Please ass assert if it is True
line 30 Remove TODO
line 32 Does using input.get_shape break reshapablity? Could we create ShapeOf subgraph instead?